README.md

```diff
@@ -110,15 +142,20 @@ For example in Docker:
 $ docker run --rm -e SPLUNK_ACCESS_TOKEN=12345 -e SPLUNK_BALLAST_SIZE_MIB=683 \
     -e SPLUNK_REALM=us0 -p 13133:13133 -p 14250:14250 -p 14268:14268 -p 55678-55680:55678-55680 \
     -p 6060:6060 -p 7276:7276 -p 8888:8888 -p 9411:9411 -p 9943:9943 \
-    -v collector.yaml:/etc/collector.yaml:ro \
     --name otelcol quay.io/signalfx/splunk-otel-collector:0.1.0 \
         --log-level=DEBUG
 ```
 
+> Use `--help` to see all available CLI arguments.
+
 ### Custom Configuration
 
 In addition to using the default configuration, a custom configuration can also
-be provided.
+be provided. This can be done via the `SPLUNK_CONFIG` environment variable as
+well as the `--config` command line argument.
+
+> Command line arguments take precedence over environment variables. This
+> applies to `--config` and `--mem-ballast-size-mib`.
```

Comment:
```suggestion
Command line arguments take precedence over environment variables. This
applies to `--config` and `--mem-ballast-size-mib`.
```

Reply:
I prefer for call outs to be clear to see in the readme and thus the blockquote